### ISSUE TITLE
Search Behavior

### ISSUE THREAD

**Issue by BasaraABC** [abantecart/abantecart-src#1612]
source https://forum.abantecart.com/index.php?topic=10356.0#msg39929
an inconstancy with search thought it was my store but can recreate it on the DEMO site. If you are on the search page and search for something with no results, update search parameter with something that should return results nothing is returned.
Steps to reproduce in Demo Store:
1) Search for Shoe in upper right corner and press enter, results returned as expected
2) Search for Test and no results returned on search page
3) Search for Shoes from search page no results returned.

Basically anytime we have no results returned and update the search nothing is returned.
